supervisor/api/utils.py

```diff
@@ -139,7 +139,7 @@ def api_return_error(
     if error and not message:
         message = get_message_from_exception_chain(error)
         if check_exception_chain(error, DockerAPIError):
-            message = format_message(message)
+            message = await format_message(message)
```

Comment:
`format_message` is no longer a coroutine.

```suggestion
            message = format_message(message)
```